acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/account/AccountManagementAcceptanceTest.java

```diff
@@ -16,22 +16,23 @@
 
 import tech.pegasys.ethsigner.tests.AcceptanceTestBase;
 
+import java.io.IOException;
 import java.util.List;
 
 import org.junit.Test;
 
 public class AccountManagementAcceptanceTest extends AcceptanceTestBase {
 
   @Test
-  public void ethSignerAccountListHasSingleEntry() {
+  public void ethSignerAccountListHasSingleEntry() throws IOException {
```

Comment:
Is this Exception actually needed?

Reply:
This one is needed

Reply:
After fixing the Accounts and Transactions classes this is no longer needed!